### PR TITLE
Implemented functionality for find order page ( #1wn444v )

### PR DESCRIPTION
**Pull request by meet-aniket**
This branch was deleted from my forked repo and I was not able to make changes in PR which were recommended by @ymaheshwari1 Sir so I am making a new pull request for this change and closing that pull request,

### CODE REVIEW

File: src/views/Orders.vue [hunk 1]
```diff
@@ -277,7 +277,40 @@ export default {
     IonTitle,
     IonToolbar
   },
+  computed: {
+    ...mapGetters({
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:281]: Import mapGetters first.

File: src/views/Orders.vue [hunk 2]
```diff
@@ -277,7 +277,40 @@ export default {
...
+      }).then(() => {
+        showToast(this.$t('Copied', { text }));
+      })
+    },
+  },
+  mounted() {
+    this.getOrders();
+  },
   setup() {
+    const store = useStore();
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:313]: Import store first.

File: src/views/Orders.vue [hunk 3]
```diff
@@ -277,7 +277,40 @@ export default {
...
+        groupByField: 'orderId',
+        filters: JSON.parse(process.env.VUE_APP_ORDER_FILTERS)
+      }
+      await this.store.dispatch("order/findOrders", payload)
+    },
+    async copyToClipboard(text: any) {
+      await Clipboard.write({
+        string: text
+      }).then(() => {
+        showToast(this.$t('Copied', { text }));
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:305]: Import showToast first.

File: src/views/Orders.vue [hunk 4]
```diff
@@ -277,7 +277,40 @@ export default {
...
+        viewSize: viewSize,
+        queryString: '* *',
+        groupLimit: 10000,
+        groupByField: 'orderId',
+        filters: JSON.parse(process.env.VUE_APP_ORDER_FILTERS)
+      }
+      await this.store.dispatch("order/findOrders", payload)
+    },
+    async copyToClipboard(text: any) {
+      await Clipboard.write({
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:302]: Import Clipboard first.

import { Plugins } from '@capacitor/core';
const { Clipboard } = Plugins;

File: src/views/Orders.vue [hunk 5]
```diff
@@ -277,7 +277,40 @@ export default {
...
   },
+  computed: {
+    ...mapGetters({
+      orders: 'order/getList',
+      getProduct: 'product/getProduct',
+      currentFacilityId: 'user/getCurrentFacility'
+    })
+  },
+  methods: {
+    async getOrders(vSize?: any, vIndex?: any){
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:288]: Please confirm if you have used <script lang="ts"> instead of <script> as you are also using typescript here.

**Comment by bashu22tiwari** [comment]
@meet-aniket 
1. As you have changed name of Order.vue to Orders.vue. Please check if you have also updated it in router.
2. Also, now we do not push .env file, so you can also remove it.
3. The name of PR says that you wanted to make the find order page functional, but you have made the orders page functional. WDYT ?

File: src/views/Orders.vue [hunk 6]
```diff
@@ -292,8 +325,8 @@ export default {
       ticketOutline,
       timeOutline
     };
-  },
-};
+  }
+})
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:329]: Remove the extra bracket.

File: src/store/modules/order/actions.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    try {
+      resp = await OrderService.findOrder(payload)
+      if (resp && resp.status === 200 && !hasError(resp)) {
+        const orders = resp.data.grouped.orderId;
+        // Add some product data into cache to access it later 
+        this.dispatch('product/getProductInformation', { orders });
+
+        if (payload.viewIndex && payload.viewIndex > 0) orders.groups = state.list.items.concat(orders.groups)
+        commit(types.ORDER_LIST_UPDATED, {
+          items: orders.groups,
```
Comment (bashu22tiwari) [inline src/store/modules/order/actions.ts:23]: Improve indentation.

File: src/main.ts [hunk 8]
```diff
@@ -49,6 +49,15 @@ app.config.globalProperties.$filters = {
     // TODO Fix this setDefault should set the default timezone instead of getting it everytiem and setting the tz
     return moment.utc(value, inFormat).tz(userProfile.userTimeZone).format(outFormat ? outFormat : 'MM-DD-YYYY');
   },
+  getOrderIdentificationId(identifications: any, id: string) {
```
Comment (bashu22tiwari) [inline src/main.ts:52]: If we have not used this utility anywhere, then why we are implementing it in this PR?

File: src/main.ts [hunk 9]
```diff
@@ -57,6 +66,26 @@ app.config.globalProperties.$filters = {
       featureValue = featureSplit[2] ? featureSplit[2] : '';
     }
     return featureValue;
+  },
+  getCustomerLoyalty(orderNotes: any, cusotmerLoyaltyOptions: any) {
```
Comment (bashu22tiwari) [inline src/main.ts:70]: If we have not used this utility anywhere, then why we are implementing it in this PR?

File: src/main.ts [hunk 10]
```diff
@@ -57,6 +66,26 @@ app.config.globalProperties.$filters = {
...
+    if (orderNotes && cusotmerLoyaltyOptions) {
+      for (const customerLoyaltyOption of Object.entries(cusotmerLoyaltyOptions)) {
+          if (orderNotes.includes(customerLoyaltyOption[0])) {
+            customerLoyalty = customerLoyaltyOption[1];
+          }
+      }
+    }
+    return customerLoyalty;
+  },
+  getShippingFrom(orderRoles: any, shippingOption: any){
```
Comment (bashu22tiwari) [inline src/main.ts:81]: If we have not used this utility anywhere, then why we are implementing it in this PR?

File: .env.example [hunk 11]
```diff
@@ -1,4 +1,6 @@
 VUE_APP_I18N_LOCALE=en
 VUE_APP_I18N_FALLBACK_LOCALE=en
 VUE_APP_CACHE_MAX_AGE=3600
-VUE_APP_VIEW_SIZE=10
\ No newline at end of file
+VUE_APP_BASE_URL=https://dev-hc.hotwax.io/api/
```
Comment (ymaheshwari1) [inline .env.example:4]: We won't push this base URL change, remove this base URL change.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**